Arrowgene.Ddon.GameServer/Quests/QuestStateManager.cs

```diff
@@ -582,34 +582,38 @@ protected PacketQueue SendWalletRewards(DdonGameServer server, GameClient client
                 {
                     case ExpType.ExperiencePoints:
                         modifier = server.Setting.GameLogicSetting.ExpModifier;
+                        amount = (uint)(amount * modifier);
```

Comment:
I don't think this code needs to be moved into the switch statement? The switch statement was just for determining the modifier amount and then performing the calculation one time  (without duplicating the statements).

You should be able to add these new settings to the original if statement that you removed instead.